### ISSUE TITLE
feature: add github-flavoured emojis

### ISSUE THREAD

**Issue by ryanseys** [jdan/cleaver#102]
Would be awesome to be able to type `:heart:` and have a :heart: show up!

**Comment by jdan** [comment]
Good idea. I'm still playing around with the idea of "plugins" but I haven't gotten very far. I feel like this would fit pretty nicely!
